config/projects.json

```diff
@@ -45,5 +45,6 @@
     "runtime": "typescript",
     "service": "virtual-machine",
     "type": "service"
-  }
+  },
+  "order": ["architecture", "common-library-typescript", "block-storage-service-typescript", "chatter-service-typescript", "consensus-service-typescript", "gossip-service-typescript", "public-api-service-typescript", "state-storage-service-typescript", "transaction-pool-service-typescript", "virtual-machine-service-typescript"]
```

Comment:
Can you break it to multilines, for readability?

Reply:
Done.